app/developers/access_routes.py

```diff
@@ -0,0 +1,14 @@
+from flask import Blueprint, render_template
+from flask.typing import ResponseReturnValue
+
+from app.common.auth.decorators import is_platform_admin
+from app.common.data import interfaces
+
+developers_access_blueprint = Blueprint("access", __name__, url_prefix="/access")
+
+
+@developers_access_blueprint.get("/grants")
+@is_platform_admin
+def grants_list() -> ResponseReturnValue:
+    grants = interfaces.grants.get_all_grants_by_user(interfaces.user.get_current_user())
```

Comment:
I wonder if we want to move get current user into a separate 🤏 helper as its the one thing in all of the interfaces that doesn't actually talk to the DB - a developer should reasonably assume here that we're fetching the grants from the db based on a user we're getting from the db 

This isn't anything to do with/ shouldn't be changed here